dockstore-webservice/src/main/java/io/dockstore/webservice/languages/LanguageHandlerInterface.java

```diff
@@ -260,6 +265,23 @@ default String getURLFromEntry(String dockerEntry, ToolDAO toolDAO) {
         return url;
     }
 
+    /**
+     * Resolves a relative path based on an absolute parent path
+     * @param parentPath Absolute path to parent file
+     * @param importPath Relative path the parent file
+     * @return Absolute version of relative path
+     */
+    default String convertImportPathToAbsolutePath(String parentPath, String importPath) {
```

Comment:
importPath is the name of the variable, but a relative path is what is actually is, so `convertRelativePathToAbsolutePath` might be more descriptive